torch/optim/nadam.py

```diff
@@ -1,4 +1,5 @@
 # mypy: allow-untyped-defs
+r"""NAdam algorithm."""
```

Comment:
how does this render? where does this render to?

Reply:
This seems ignored in the documentation (unless I missed anything) but takes care of docstring error.